docs/guides/patterns/knativetrigger.md

```diff
@@ -0,0 +1,179 @@
+          - containerPort: 80
+```
+
+Apply this to the cluster:‍‍
+
+```sh
+kubectl apply -f knative-service.yaml
+```
+
+Check that the Knative service is running with the following command:‍‍
+
+```sh
+kubectl get ksvc hello-http 
+NAME         URL                                          LATESTCREATED     LATESTREADY       READY   REASON
+hello-http   http://hello-http.default.svc.cluster.local  hello-http-00002  hello-http-00002  True  
+```
+
+Now deploy a TriggerMesh broker to the cluster. The broker decouples event producers and consumers, and provides triggers that send events to consumers if they match a given filter. Create the following broker.yaml:‍‍
+
+```yaml
+apiVersion: eventing.triggermesh.io/v1alpha1
+kind: MemoryBroker
+metadata:
+  name: broker
+```
+
+And apply it to the cluster:‍‍
+
+```sh
+kubectl apply -f broker.yaml
+```
+
+Check that the broker is available:‍‍
+
+```sh
+kubectl get memorybrokers.eventing.triggermesh.io 
+NAME     URL                                                 AGE     READY   REASON
+broker   http://broker-mb-broker.default.svc.cluster.local   3h38m   True    
+```
+
+Now create a Webhook source, in a file called webhook-source.yaml, that we'll use as an event producer. It will let us easily send events of type `my.event.type` to the broker over HTTP. Later, you can replace it or add additional event sources like Kafka, AWS SQS, Google Pub/Sub, etc… ‍‍
+
+```yaml
+apiVersion: sources.triggermesh.io/v1alpha1
+kind: WebhookSource
+metadata:
+  name: sample
+spec:
+  eventType: my.event.type
+  sink:
+    ref:
+      apiVersion: eventing.triggermesh.io/v1alpha1
+      kind: MemoryBroker
+      name: broker
+```
+
+And create the Webhook source
+
+```sh
+kubectl apply -f webhook-source.yaml
+```
+
+Create the HTTP target in a http-target.yaml file, and set its endpoint parameter to the cluster local address of the Knative service that you created before.‍‍
+
+```yaml
+apiVersion: targets.triggermesh.io/v1alpha1
+kind: HTTPTarget
+metadata:
+  name: http-target
+spec:
+  endpoint: http://hello-http.default.svc.cluster.local
+  method: GET
+```
+
+Create the HTTP target:
+
+```sh
+kubectl apply -f http-target.yaml
+```
+
+‍‍Now create a Trigger that will route events of type `my.event.type` (those produced by the Webhook source) to the HTTP target, in a trigger.yaml file:
+
+```yaml
+apiVersion: eventing.triggermesh.io/v1alpha1
+kind: Trigger
+metadata:
+  name: trigger-knative-service
+spec:
+  broker:
+    group: eventing.triggermesh.io
+    kind: MemoryBroker
+    name: broker
+  filters:
+  - exact:
+      type: my.event.type
+  target:
+    ref:
+      apiVersion: targets.triggermesh.io/v1alpha1
+      kind: HTTPTarget
+      name: http-target
+```
+
+Create the Trigger:‍‍
+
+```sh
+kubectl apply -f trigger.yaml
+```
+
+## Test it out!
+
+Run minikube tunnel in a separate terminal, so that the load balancer service is assigned an IP. That loadbalancer IP is used by the sslip.io config. 
```

Comment:
```suggestion
Run `minikube tunnel` in a separate terminal, so that the load balancer service is assigned an IP. That loadbalancer IP is used by the sslip.io config. 
```